sources/packages/web/src/components/generic/EmptyTableTemplate.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div v-if="isEmpty">
```

Comment:
since you have created this generic component, just a minor question, are we excepting more future requirements using this component?

Reply:
As per my conversation with Lynn the idea is to have an empty template to display with DataTables.
Using this component it will be just a matter to copy the image from Figma and consume it.